src/components/TodoList.js

```diff
@@ -16,4 +17,8 @@ function TodoList({ todoList, onRemoveTodo }) {
   );
 }
 
+TodoList.propTypes = {
+  todoList: PropTypes.func
```

Comment:
Similarly here, the `todoList` is an array and `onRemoveTodo` is a function
```suggestion
todoList: PropTypes.array,
onRemoveTodo: PropTypes.func.isRequired,
```